library/Rules/Image.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of Respect/Validation.
+ *
+ * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ *
+ * For the full copyright and license information, please view the "LICENSE.md"
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Rules;
+
+class Image extends AbstractRule
+{
+    public function validate($input)
+    {
+        return @getimagesize($input) ? true : false;
```

Comment:
I've put the the *if* statement this way but maybe it would be better to use Respect rules for both conditions...

[Url](https://github.com/Respect/Validation/blob/master/library/Rules/Url.php)
[File](https://github.com/Respect/Validation/blob/master/library/Rules/File.php)